index.html

```diff
@@ -92,10 +104,39 @@ <h5 class="titles">Astronaut, Policeman bablabla</h5>
             <p class="desc">blablablablbalbablablbalbalbabalbalabalbalbalbalbalbabalba</p>
           </div>
         </li>
-      </template>   
+      </template>
+    </ul>
+    <button class="btn-2" id="toggle-speaker">SHOW MORE</button>
+  </section>
+
+  <section class="partners">
+    <h3 id="partners" class="partners-h3">
+      Partner
+    </h3>
+    <hr>
+    <ul class="partners-logos">
+      <li>
+        <div class="logo-1"></div>
+      </li>
+      <li>
+        <div class="logo-2"></div>
+      </li>
+      <li>
+        <div class="logo-3"></div>
+      </li>
+      <li>
+        <div class="logo-4"></div>
+      </li>
+      <li>
+        <div class="logo-5"></div>
+      </li>
     </ul>
   </section>
 
+  <footer>
+    <div class="event-logo"></div>
+    <p class="footer-p"> 2021 Digital arts. None Rigths Reserved </p>
+  </footer>
   <!-- <div class="icon-1"></div> -->
```

Comment:
- [ ] You left a comment in a pushed code to the repository. Let's ride all comments in the code version hosted on the repository to enhance cleaner code.

Reply:
Thanks already on it!